docs/submitting-bulk-sentences.md

```diff
@@ -1,73 +1,49 @@
 All the sentences read by our contributors are sourced from copyright free sources or permissioned sources, either through [automated sourcing](https://github.com/common-voice/cv-sentence-extractor) by agreement from sources like Wikipedia or contributed by our language communities.
 
-If you want to add a single sentence, or a series of single sentences, you can do so via the [Sentence Collector](https://foundation.mozilla.org/en/blog/the-new-common-voice-sentence-collector/). But when you have a large volume of sentences to contribute, you can make a bulk sentence submission using GitHub.
+If you want to add a single sentence, or a series of single sentences, you can do so via the [Sentence Collection](https://commonvoice.mozilla.org/write) page on the [Common Voice website](https://commonvoice.mozilla.org). To contribute a larger number of sentences at once, you can use the Bulk Sentence upload option.
 
 Remember that for both single sentence and bulk sentence submissions, sentences must:
 - Be in the public domain, with a CC0 license
 - Be short, readable and take about 10-15 seconds to read
 - Avoid including numbers or special characters
+## Formatting Your Bulk Sentences
+To upload bulk sentences, you'll need to have created a TSV file with your sentences.
 
-Please format your bulk sentences into a TSV file with two columns, one containing the sentences you would like to submit and the other containing the source of the sentences.
+Please format your bulk sentences into a [.TSV](https://en.wikipedia.org/wiki/Tab-separated_values) file with five columns, containing the following columns from left to right:
+- Your sentence
+- The source for your sentence
+- Additional infomation about why this source is eligible for inclusion in a CC0 dataset
+- A blank column for our team to use in the quality control process
+- An optional column showing the domain for sentences, as described in this blog post
```

Comment:
Should there be a link to the blog post for the sentence domain in this sentence?